src/main/java/org/GehtSoftHWByAziz/HW7/RESTCustomWebServer.java

```diff
@@ -1,21 +1,31 @@
 package org.GehtSoftHWByAziz.HW7;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.GehtSoftHWByAziz.HW6.Task2.CustomWebServer;
 import org.GehtSoftHWByAziz.HW7.RESTAnnotations.*;
 
+import java.io.PrintWriter;
 import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class RESTCustomWebServer extends CustomWebServer {
-  private Map<String, Method> httpControllersMethods = new HashMap<>();
+  private final Map<String, Method> httpControllersMethods = new HashMap<>();
```

Comment:
accepted for now, however, serving requests a link the dispatching of them to particular handlers are different responsibilities (thus must be separated from each other)